dw-dialog.js

```diff
@@ -281,62 +170,79 @@ export class DwDialog extends LitElement {
     this._onDialogScroll = this._onDialogScroll.bind(this);
   }
 
-  connectedCallback() {
-    super.connectedCallback && super.connectedCallback();
-    this.updateComplete.then(() => {
-      this._initDialog();
-      this._listenEvents();
-      this._checkAvailableSlot();
-    })
-  }
-
   disconnectedCallback() {
+    if (this.type !== 'modal') {
+      super.disconnectedCallback();
```

Comment:
@nirmalbaldaniya-dream हमे यहा पे 2 बार `super.disconnectedCallback();` नहीं लिखना चाहिए, `if (this.type === 'modal')` हे तब हमें निचे का सब काम करना हे तो यही condition में निचे का काम करके common  `super.disconnectedCallback();` single time लिखना चाहिए